src/h5web/toolbar/controls/ToggleGroup.tsx

```diff
@@ -53,7 +54,7 @@ function Btn(props: BtnProps) {
     >
       <span className={styles.btnLike}>
         {Icon && <Icon className={styles.icon} />}
-        <span className={styles.label}>{label}</span>
+        {!iconOnly && <span className={styles.label}>{label}</span>}
```

Comment:
You also need to add an `aria-label` to the button, so that it remains accessible (like in `Btn`)